.vscode/settings.json

```diff
@@ -4,5 +4,16 @@
     "esversion": 6,
     "node": true
   },
-  "typescript.tsdk": "./node_modules/typescript/lib"
+  "typescript.tsdk": "./node_modules/typescript/lib",
+  "search.exclude": {
+    "**/coverage": true,
+    "**/dist": true
+  },
+  "files.watcherExclude": {
+    "**/dist/**": true
+  },
+  "files.exclude": {
+    "**/dist": true,
+    "**/node_modules": true
+  }
 }
```

Comment:
I agree with you, I feel like editor configurations are particular to each dev.  

Since it was already checked in, I added something I thought was helpful.

But I think  we should add the file to `.gitIgnore`

Reply:
Fixed in e49ed34